docs/source/migrating.rst

```diff
@@ -0,0 +1,189 @@
+Migrating from IPython
+======================
+
+`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+has changed a few things, moving the various language-agnostict
+components of IPython under the Jupyter umbrella. This doc should help
+you find what's changed, and how to keep doing what you've been doing.
+
+Where have all my files gone?
+-----------------------------
+
+AKA Why isn't my configuration having any effect anymore?
+
+Jupyter splitting out from IPython means that the locations of some
+files have moved, and Jupyter projects have not inherited everything
+from how IPython did it.
+
+The ``jupyter migrate`` command copies several of these files to their
+new homes.
+
+Quick reference
+~~~~~~~~~~~~~~~
+
+A quick reference for common locations of relocated files.
+
+Configuration files
+^^^^^^^^^^^^^^^^^^^
+
```

Comment:
I would make it clear if the user need to manually copy these or if `jupyter migrate` will